uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,41 @@
+#
+# Raw Air Data.
+#
+
+# Note: unused vars should be assigned NaN
+
+#
+# Heater State
+# 
+uint8 FLAG_HEATER_AVAILABLE      = 0
+uint8 FLAG_HEATER_WORKING        = 1
+uint8 FLAG_HEATER_OVERCURRENT    = 2
+uint8 FLAG_HEATER_OPENCIRCUIT    = 3
+uint8 flags
```

Comment:
Hah, good catch, I entirely overlooked it. Of course, the current values won't work.